### PR TITLE
Skip Installation on Travis

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by turbaszek — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -149,6 +149,7 @@ jobs:
         INSTALL_AIRFLOW_VERSION="1.10.6"
       stage: test
       script: ./scripts/ci/ci_prepare_and_test_backport_packages.sh
+install: skip
```
Comment (kaxil) [inline .travis.yml:152]: sry, what are we skipping here?

Comment (potiuk) [reply .travis.yml:152]: Sorry that was quick. I had no time for explanation. I realized our build started to take long time.... and it turned out Travis is trying to be smart and when it sees requirements.txt it tries to install it automatically in the host...

That's what we are skipping :).

**Review by kaxil — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (kaxil) [reply .travis.yml:152]: got it

Comment (potiuk) [reply .travis.yml:152]: :man_facepalming:

**Pull request merged**